spec/features/post_an_ad_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe "Ad posting", type: :system do
+  before do
+    county = County.create!(name: "SISAČKO-MOSLAVAČKA")
+    City.create!(name: "Petrinja", county: county)
+    City.create!(name: "Glina", county: county)
```

Comment:
Should we add factory bot or you think it won't be needed...or you prefer not to use it altogether?

Reply:
I prefer to use it, I think it will probably be needed but I opted to skip that to get this working faster. We can easily add it in the next tests MR.

Reply:
👍 for factory bot.

Reply:
Also, I think that (default) counties and cities should be added to fixtures so they are always there. We add all counties and one or two biggest cities in it.

Reply:
I realised that we already have seeds which run with `RAILS_ENV=test bin/rails db:setup` which is I how I set it up in the CI script. I also then realised seeds are broken so I fixed them by moving this count and city setup code there. 

In the spirit of simplicity I'll stick with that instead of fixtures, at least for now. It also has the benefit of tests ensuring our seeds are working.